spec/models/initiative_spec.rb

```diff
@@ -26,4 +26,19 @@
     expect{ delete :destroy, params: {id: initiative.id}}.to change(Initiative,:count).by(-1)
     expect(response).to redirect_to root_path
   end
+end
+
+describe "#index" do
```

Comment:
Now, I know this PR didn't add this `InitiativesController` block, but since you're adding behavior that tests the controller, I think this would be a good opportunity to move the whole block to its own file at `spec/controllers/initiative_controller_spec.rb`. This file is no longer just a model spec, and that could make the controller specs hard to find when we need to go in and add more tests later. Separating model and controller specs into their own files is a Rails convention that comes from the idea of "separation of concerns", a key principle of object oriented programming.

Reply:
Moved the block to its own file